bevy_script_api/src/lua/util.rs

```diff
@@ -217,3 +229,7 @@ macro_rules! impl_tealr_generic{
 // }
 
 // pub(crate) use impl_user_data;
+
+#[derive(Resource, Default, Clone, Reflect)]
+#[reflect(Resource)]
+pub struct TypeRegistryWrapper(#[reflect(ignore)] pub TypeRegistry);
```

Comment:
Do we need a wrapper? I think the type registry we should use is now [AppTypeRegistry](https://docs.rs/bevy/latest/bevy/app/struct.AppTypeRegistry.html#) or would that be inappropriate?

Reply:
I was unsure how things worked, so I just did the simplest thing that worked without changing much else. I am not familiar with Reflect.

Reply:
I replaced all of the wrapper types with AppTypeRegistry in my recent commit. It shows up bad since generate.rs doesn't exist.

Reply:
Are you sure you pushed that commit? I cannot see relevant changes on here

Reply:
Ah, I may have done it with another commit, and then reverted all the changes. I'll do it soon.